pcl_ros/CMakeLists.txt

```diff
@@ -87,8 +87,7 @@ generate_dynamic_reconfigure_options(
 catkin_package(
   INCLUDE_DIRS include
   LIBRARIES
-    pcl_ros_filters
```

Comment:
Both pcl_ros_filters and pcl_ros_io have their symbols stripped, so it makes no sense to export these libraries.